src/meta-srv/src/keys.rs

```diff
@@ -228,6 +228,21 @@ pub struct StatValue {
     pub stats: Vec<Stat>,
 }
 
+impl StatValue {
+    /// Get the region number from stat value.
+    pub fn region_num(&self) -> Option<u64> {
+        for stat in self.stats.iter() {
+            let region_num = stat.region_num;
+            if region_num < 0 {
```

Comment:
How can region number became negative?

Reply:
`region number < 0` indicates that an error occurred when the datanode obtained the region number.

I will add some comment here.

Reply:
Why not use Option?